src/js/components/Select/Select.js

```diff
@@ -278,19 +278,37 @@ const Select = forwardRef(
 
     const iconColor = getIconColor(theme);
 
+    let ariaLabelGenerated = '';
+    if (value) {
+      if (Array.isArray(value)) {
+        ariaLabelGenerated = `; ${value.length || 0} ${
```

Comment:
Why aren't we using `format()` for all of these cases? As shown, if the `value` is an Array, we won't be using the message catalog to localize the aria label. I was expecting we would introduce more keys in the catalog for the various permutations, allowing simple localization of each permutation.